### ISSUE TITLE
Move Virtual Server: PHP-FPM listen socket gets parent server id

### ISSUE THREAD

**Issue by iliaross** [virtualmin/virtualmin-gpl#610]
Jamie, there is a bug with currently checked in code.

I had two top-level virtual servers running different PHP versions.

Then I moved top-level `server2` under `server1` using **Server Configuration ⇾ Move Virtual Server** everything seemed just fine at first.

Later, I went to **PHP Options** page for the parent `server1` and tried to change PHP version and received an error:

<img width="1412" alt="image" src="https://github.com/virtualmin/virtualmin-gpl/assets/4426533/138da0e7-d46a-42be-a226-f11e6cfc69af">

So, when top level `server2` was placed/moved as sub-server under `server1`, the `server2` PHP-FPM config received the socket id with the parent `server1` and not `server2`.

For example:

parent `server1` - 16917790583820
subsrv `server2` - 167390864779391

After the move `server2` got in PHP-FPM config:

```
listen = /var/php-fpm/16917790583820.sock
```

.. when it should have been:

```
listen = /var/php-fpm/167390864779391.sock
```

**Comment by jcameron** [comment]
That's an odd one, because the domain ID shouldn't change when changing ownership.

Is this system running Apache or Nginx?

**Comment by iliajie** [comment]
> Is this system running Apache or Nginx?

It was Apache.

> That's an odd one, because the domain ID shouldn't change when changing ownership.

The domain ID is changed but not in `listen` socket line in PHP-FPM config.

**Comment by jcameron** [comment]
But moving a domain under a new parent doesn't change the domain ID!

**Comment by iliajie** [comment]
> But moving a domain under a new parent doesn't change the domain ID!

You're right! This is very irritating. I can no longer reproduce this issue. I will do more digging a bit later and let you know.

**Comment by jcameron** [comment]
Ok, please let me know if you find steps to repro this..

**Comment by iliaross** [comment]
Reopened, as this was closed by an automated cleanup run.

⚠️ If the issue is resolved, please take a moment to close it to help keep the project organized.
